### ISSUE TITLE
关于尺寸不匹配报错

### ISSUE THREAD

**Issue by DuowenC** [Linwei-Chen/FreqFusion#22]
感谢您的工作！
请问您的方法mask2former在ADE数据集上测试时resize的尺寸是怎么设置的？我看了对应的config也没能解决。会在您的freqfusion.py中233行：

mask_lr_lr_feat = F.interpolate( #利用初始低通滤波对另一部分初始低通滤波特征上采样
                        carafe(mask_lr_lr_feat_lr, mask_lr_init, self.lowpass_kernel, self.up_group, 2), size=compressed_hr_feat.shape[-2:], mode='nearest')

上述地方报错，可能是由于mask_lr_lr_feat_lr（尺寸为1，25，14，10）和 mask_lr_init （1，25，27，20）的14和27尺寸不是正好的二倍，从而carafe出现assertion error：assertion masks.size(-2) == features.size(-2) * scale_factor。

**Comment by DuowenC** [comment]
我排查了一下可能还是因为mask2former对于某些尺寸的图像，如果不是64的倍数，可能出现res4和res5的图像尺寸不是正好二倍的关系，从而在后续fusion时出现问题。因此想问一下您是怎么设置ADE测试的图像尺寸大小的~~

**Comment by Linwei-Chen** [comment]
您可以参考：
https://github.com/Linwei-Chen/FreqFusion/issues/5
